quickshop-bukkit/src/main/java/com/ghostchu/quickshop/localization/text/SimpleTextManager.java

```diff
@@ -415,7 +416,7 @@ public ReloadResult reloadModule() {
   public ProxiedLocale findRelativeLanguages(@Nullable final String langCode) {
     //langCode may null when some plugins providing fake player
     if(langCode == null || langCode.isEmpty()) {
-      return new ProxiedLocale(langCode, DEFAULT_LOCALE, NumberFormat.getCompactNumberInstance(Locale.US, NumberFormat.Style.SHORT), Locale.ROOT);
+      return new ProxiedLocale(langCode, DEFAULT_LOCALE, getCompactNumberInstance(Locale.US), Locale.ROOT);
```

Comment:
Should have blank line above here per the updated style guidelines.

Reply:
fixed